.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846
+  // for the documentation about the extensions.json format
+  "recommendations": [
```

Comment:
GitHub don't like Json comments. I see https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

Should we remove the comments or convert in another form?

Reply:
Perhaps but it didn't stop the VSCode team from checking json with comments: https://github.com/Microsoft/vscode/blob/master/.vscode/extensions.json
https://github.com/Microsoft/vscode/blob/master/tslint.json

And that comment is the default comment added when you run the `Extensions: Configure Recommended Extensions` command in VSCode.

Reply:
If VS Code understands it, I'll merge.